### PR TITLE
[Merged by Bors] - Schedule gnosis merge

### PR DESCRIPTION
**Pull request by dapplion**
## Issue Addressed

N/A

## Proposed Changes

Schedule Gnosis merge
- Upstream config PR: https://github.com/gnosischain/configs/pull/3
- Nethermind PR: https://github.com/NethermindEth/nethermind/pull/4901
- Public announcement: https://twitter.com/gnosischain/status/1592589482641223682

## Additional Info

N/A

### CODE REVIEW

**Review by michaelsproul — CHANGES_REQUESTED** [review]
We also need to update the `ChainSpec::gnosis` function to match.

You can run `cargo test --release` in `common/eth2_network_config` to check everything matches

**Comment by michaelsproul** [comment]
Hmm, seems I was wrong. There's no test in `eth2_network_config` that catches this :frowning:. I think there might be a preset test, but that's unlikely to be sufficient.

I'll fix this up and push to your branch if you don't mind?

**Comment by dapplion** [comment]
> Hmm, seems I was wrong. There's no test in `eth2_network_config` that catches this frowning. I think there might be a preset test, but that's unlikely to be sufficient.
> 
> I'll fix this up and push to your branch if you don't mind?

Yes please, take over the branch, thank you!

File: consensus/types/src/chain_spec.rs [hunk 1]
```diff
@@ -777,7 +777,7 @@ impl ChainSpec {
             domain_sync_committee_selection_proof: 8,
             domain_contribution_and_proof: 9,
             altair_fork_version: [0x01, 0x00, 0x00, 0x64],
-            altair_fork_epoch: Some(Epoch::new(256)),
+            altair_fork_epoch: Some(Epoch::new(512)),
```
Comment (michaelsproul) [inline consensus/types/src/chain_spec.rs:780]: Funnily enough this value was already wrong in the `ChainSpec::gnosis` function, which I think indicates that we don't really use these values. I think we'll always (?) override them with the ones from the config.yaml.

Reference for 512 being the correct val: https://github.com/gnosischain/configs/blob/6af0c780f531358ff285cd2807915f1c4caca9ab/mainnet/config.yaml#L150

Comment (michaelsproul) [reply consensus/types/src/chain_spec.rs:780]: Removing this level of redundancy is out of scope for this PR, so I think we may as well just set them all to the right values

**Comment by michaelsproul** [comment]
Confirmed that this works on my Gnosis node (with Nethermind's `master` branch) :sunglasses:

> Nov 18 05:36:37.503 INFO Ready for the merge                     current_difficulty: 8519301852020543110622850477420385442753109103, terminal_total_difficulty: 8626000000000000000000058750000000000000000000, service: slot_notifier

Will get one more sanity check from a Lighthouse dev then we can merge :tada:

**Review by michaelsproul — APPROVED** [review]

**Review by paulhauner — APPROVED** [review]
Looks good to me! Let's mergeeee 🎉  Props to @dapplion for being the Danny Ryan of Gnosis ❤️

**Pull request closed without merging**